# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mesa-legacy/hotfix-no-shared.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mesa/include/meson.build.vanilla	2025-07-28 23:24:12.248380143 +0200
+++ mesa/include/meson.build	2025-07-28 23:26:05.938715749 +0200
@@ -90,7 +90,2 @@
 # Non-upstream headers
 if with_egl
-  install_headers(
-    'EGL/eglmesaext.h',
-    'EGL/eglextchromium.h',
-    subdir : 'EGL',
-  )
@@ -101,7 +100,6 @@
 endif
 
 if with_dri
-  install_headers('GL/internal/dri_interface.h', subdir : 'GL/internal')
 endif
 
 if with_gallium_st_nine
--- mesa/src/mesa/drivers/dri/meson.build.vanilla	2025-07-28 23:41:13.823970079 +0200
+++ mesa/src/mesa/drivers/dri/meson.build	2025-07-28 23:41:45.650559477 +0200
@@ -84,13 +84,4 @@
   if dep_libdrm.found()
     dri_req_private = ['libdrm >= ' + _drm_ver]
   endif
-
-  pkg.generate(
-    name : 'dri',
-    filebase : 'dri',
-    description : 'Direct Rendering Infrastructure',
-    version : meson.project_version(),
-    variables : ['dridriverdir=' + dri_drivers_path],
-    requires_private : dri_req_private,
-  )
 endif
--- mesa/src/util/meson.build.vanilla	2025-07-29 13:16:55.299508246 +0200
+++ mesa/src/util/meson.build	2025-07-29 13:17:16.762387649 +0200
@@ -152,7 +152,7 @@
 
 files_drirc = files('00-mesa-defaults.conf')
 
-install_data(files_drirc, install_dir : join_paths(get_option('datadir'), 'drirc.d'))
+#install_data(files_drirc, install_dir : join_paths(get_option('datadir'), 'drirc.d'))
 
 if with_tests
   prog_xmllint = find_program('xmllint', required : false, native : true)
